src/main.rs

```diff
@@ -56,6 +67,20 @@ pub struct Arguments {
     #[arg(long, global = true)]
     generate: bool,
 
+    /// Generate combined trace until `combined_trace` since the trace starts
+    ///
+    /// If this is set, the program will only generate the csv to stdout and exit.
+    #[arg(
+        long,
+        global = true,
+        value_parser = parse_duration
+    )]
+    combined_trace: Option<Duration>,
+
+    /// Generate combined trace csv file to the specified path instead of stdout
+    #[arg(long, requires = "combined_trace", global = true, value_name = "File")]
+    combined_trace_path: Option<PathBuf>,
```

Comment:
We should let user specify the output format, start_time and end_time. For the output path, we should change the extension according to the format.